src/pages/bubbles/[number].svelte

```diff
@@ -0,0 +1,169 @@
+<div class="circles" data-cy="bubbles">
+    {#each bubbles as bubble, index}
+        <div class="bubble"
+             on:click={ (event) => pop(event, index) }
+             style="{ getStyle(bubble) }"
+             class:popping="{ bubbles[index].popping }"
+             on:animationiteration={ (event) => animationEndHandler(event, index) }
+        >
+            <div class="inner"></div>
+            <div class="particles"></div>
+        </div>
+    {/each}
+</div>
+<script>
```

Comment:
```suggestion
</div>

<script>
```
I would separate these.